dashboard/app/models/pd/enrollment.rb

```diff
@@ -17,6 +17,7 @@
 #  completed_survey_id :integer
 #  school_info_id      :integer
 #  deleted_at          :datetime
+#  properties          :text(65535)
```

Comment:
Should this be here?

Reply:
No, weird. I must have run an intermediate version of [Clare's migration](https://github.com/code-dot-org/code-dot-org/pull/22897/files#diff-b8bdc35710f298dcd7ec369834c742f2R3) at some point. Will remove